libraries/AP_HAL_ChibiOS/hwdef/PyroF4OSD/hwdef.dat

```diff
@@ -0,0 +1,153 @@
+# hw definition file for PyroDrone F4 OSD
+# WORK IN PROGRESS (NOT TESTED)
+
+# MCU class and specific type
+MCU STM32F4xx STM32F405xx
+
+# board ID for firmware load
+APJ_BOARD_ID 125
+
+# crystal frequency
+OSCILLATOR_HZ 8000000
+
+define STM32_ST_USE_TIMER 4
+define CH_CFG_ST_RESOLUTION 16
+
+FLASH_SIZE_KB 1024
+
+
+# only one I2C bus
+I2C_ORDER I2C1
+
+# order of UARTs (and USB)
+SERIAL_ORDER OTG1 USART1 UART2 USART3 UART4 UART5
+
+# LEDs
+PB9 LED_BLUE OUTPUT LOW GPIO(0)
+
+define HAL_GPIO_A_LED_PIN 0
+
+# buzzer
+PB5 BUZZER OUTPUT GPIO(80) LOW
+define HAL_BUZZER_PIN 80
+define HAL_BUZZER_ON 1
+define HAL_BUZZER_OFF 0
+
+# spi1 bus for IMU
+PA5 SPI1_SCK SPI1
```

Comment:
The most important thing to check is the pin assignments from Betaflight. If they have based this on the same board as you then maybe no changes are needed.

Reply:
They should be correct according to the betaflight target, I will test everything on my board before changing from a draft